minio/minio
# Check bucket & prefix after obj's for new bucket are fetched

## Description

When the request to list objects in a bucket concludes, a check has been introduced to see if the current bucket name and prefix are the same as when the request was made before updating the displayed objects.

## Motivation and Context


Fixes #7837

## Regression


Yes, from #7805

## How Has This Been Tested?



1. Create a bucket named `large` with 100k objects.
2. Open the Network tab on the Browser Dev Console to track requests
3. Create a bucket named `small` with a single object.
4. Click on the bucket name `large` from the sidebar to load its contents.
5. While the request is still going on (when we see the spinner and before the objects are loaded), click on the bucket name `small` to load its contents.
6. Observe that the objects corresponding to bucket named `small` are displayed instantly as the request to retrieve them doesn't take long.
7. Track the request to list the objects for the bucket named `large` and verify that the request has ended and the objects corresponding to that bucket are not loaded on top.

## Types of changes


## Checklist:
